tests/foreman/api/test_user.py

```diff
@@ -508,6 +508,33 @@ def test_negative_create_invalid_length_ssh_key(self):
             'Fingerprint could not be generated'
         )
 
+
+    @tier1
+    def test_negative_create_ssh_key_with_invalid_name(self):
+        """Attempt to add SSH key that has invalid name length
+
+        :id: e1e17839-a392-45bb-bb1e-28d3cd9dba1c 
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Attempt to add invalid ssh Key name to above user
+
+        :expectedresults: Satellite should raise Name is too long assertion
+
+        :CaseImportance: Critical
+        """
+        invalid_ssh_key_name = invalid_names_list()
```

Comment:
if you don't loop here, so better to use `gen_string` directly, e.g. `gen_string('alpha', 300)`